File: pylabs/lab2.py
import math
print('Введите числа, не равные нулю')
x = int(input('Введите значение х: '))
a = int(input('Введите значение а: '))
w = int(input('1 - вычислить функцию G, 2 - вычислить функцию F, 3 - вычислить функцию Y: '))
if w == 1:
    print('G = ' + str(9*(7*a**2-19*a*x+10*x**2)/(25*a**2+30*a*x+9*x**2)))
elif w == 2:
    print('F = ' + str(math.cos(9*a**2-13*a*x-10*x**2)))
elif w == 3:
    print('Y = ' + str(math.log(-80*a**2-46*a*x+21*x**2+1)/math.log(10)))
else:
    print('Всё плохо')

File: clabs/lab2.c
/*comment*/
#include <stdio.h>
#include <math.h>

int main(void)
{
float a,x,g,f,y;
int word;
printf("Введите х ");
scanf("%f", &x);
printf("Введите a ");
scanf("%f", &a);
scanf("%i",&word);
switch(word)
{
    case 1:
        g = 9*(7*a*a-19*a*x+10*x*x)/(25*a*a+30*a*x+9*x*x);
        printf("g=%f\n\n",g);
        break;
    case 2:
        f = cos(9*a*a-13*a*x-10*x*x);
        printf("f=%f\n\n",f);
        break;
    case 3:
        y = log(-80*a*a-46*a*x+21*x*x+1);
        printf("y=%f\n\n",y);
        break;
    default:
        printf("неправильный ввод.\n");
}

}

